README.md

```diff
@@ -316,11 +316,23 @@ interface {
 }
 ```
 
+#### [Enable Error Reporting](https://zwave-js.github.io/node-zwave-js/#/api/driver?id=enableerrorreporting)
+
+[compatible with schema version: 16+]
+
+```ts
+interface {
+  messageId: string;
+  command: "driver.enable_error_reporting";
```

Comment:
How can one disable error reporting ?

Reply:
it's now opt in. This is related to Sentry reporting, see here for more info: https://zwave-js.github.io/node-zwave-js/#/getting-started/migrating-to-v9?id=error-reporting-is-now-opt-in-and-less-disruptive

My plan is to opt in to Sentry reporting in HA if the user has opted in to data collection, otherwise to not